clang/test/Driver/linker-wrapper-llvm-help.c

```diff
@@ -0,0 +1,10 @@
+// Check that these simple command lines for listing LLVM options are supported,
```

Comment:
Do we have any other tests that just check the output for `--help`? Might be a little excessive.

Reply:
Grepping for -help in llvm's test suite finds various such tests. The point here is to make sure `--offload-opt=--help can list these opt-like options as opposed to the normal clang-linker-wrapper options.  Seems like an important feature.